src/resultspage.cpp

```diff
@@ -104,10 +104,10 @@ ResultsPage::ResultsPage(PerfParser* parser, QWidget* parent)
     m_summaryPageDock->addDockWidgetAsTab(m_disassemblyDock, KDDockWidgets::InitialVisibilityOption::StartHidden);
     m_disassemblyDock->toggleAction()->setEnabled(false);
     m_summaryPageDock->setAsCurrentTab();
-    if (m_frequencyPage) {
-        m_frequencyDock = dockify(m_frequencyPage, QStringLiteral("frequency"), tr("Fr&equency"), tr("Ctrl+E"));
-        m_summaryPageDock->addDockWidgetAsTab(m_frequencyDock);
-    }
+#if QCustomPlot_FOUND
```

Comment:
why this change?

Reply:
`frequencypage.h` is only included when `QCustomPlot` is found. If it is not found, the compiler doesn't know that `m_frequencyPage` is a QWidget and fails to compile.
This Change is also in #360

Reply:
ok but that has nothing to do with the topic of "update readme to announce annotate view" and should thus not be part of this pull request